setup-ovs.sh

```diff
@@ -67,7 +75,17 @@ if [[ $(nmcli conn | grep -c ovs) -eq 0 ]]; then
       nmcli c delete $(nmcli c show |grep ovs-cnv |awk '{print $1}') || true
   else
       nmcli conn mod brcnv-iface connection.autoconnect yes
+      nmcli conn up ovs-slave-$secondary_device
+      # ocp 4.7
+      rm -r /var/ovsbond47 || true
+      cp -r /etc/NetworkManager/system-connections-merged /var/ovsbond47 || true
+      # ocp 4.8
+      rm -r /var/ovsbond48 || true
+      cp -r /etc/NetworkManager/systemConnectionsMerged /var/ovsbond48 || true
+      #reboot
   fi
 else
     echo "ovs bridge already present"
+    for c in $(nmcli c show | grep ovs | awk '{print $1}'); do nmcli c up $c; done
+    nmcli c up brcnv-iface
```

Comment:
I think `brcnv-iface` is already included part of the former loop:
`nmcli c show` gives 
```
NAME                  UUID                                  TYPE           DEVICE     
brcnv-iface           037e93db-2394-4134-93f6-e2bfb652f998  ovs-interface  brcnv      
Wired connection 1    cab519e7-054f-38cc-a2eb-99c3d54f9c3f  ethernet       ens5       
ovs-bridge-brcnv      60f18692-cd3f-4089-a9c5-0d28ad941bc7  ovs-bridge     brcnv      
ovs-slave-bond0       f4acdfaa-2a40-4f0c-ac71-50e31555a77c  ovs-port       bond0      
ovs-slave-brcnv-port  77b5b001-6202-481a-9dda-e89b0a92e702  ovs-port       brcnv-port 
ovs-slave-veth1-host  496d3726-7eec-4d61-87b6-438dcd96fd45  ethernet       veth1-host 
ovs-slave-veth2-host  c919661c-7107-4917-98d1-d33e88852a53  ethernet       veth2-host 
Wired connection 2    27fbfbbc-7e47-3810-a617-877d888c3abc  ethernet       --         
Wired connection 3    b4687319-c9f4-3f7a-bc71-2b5e61f279ac  ethernet       --
```
piping `grep ovs` gives
```
brcnv-iface           037e93db-2394-4134-93f6-e2bfb652f998  ovs-interface  brcnv      
ovs-bridge-brcnv      60f18692-cd3f-4089-a9c5-0d28ad941bc7  ovs-bridge     brcnv      
ovs-slave-bond0       f4acdfaa-2a40-4f0c-ac71-50e31555a77c  ovs-port       bond0      
ovs-slave-brcnv-port  77b5b001-6202-481a-9dda-e89b0a92e702  ovs-port       brcnv-port 
ovs-slave-veth1-host  496d3726-7eec-4d61-87b6-438dcd96fd45  ethernet       veth1-host 
ovs-slave-veth2-host  c919661c-7107-4917-98d1-d33e88852a53  ethernet       veth2-host 
```
then `awk '{print $1}'` should produce these connections:
```
brcnv-iface
ovs-bridge-brcnv
ovs-slave-bond0
ovs-slave-brcnv-port
ovs-slave-veth1-host
ovs-slave-veth2-host
```

Reply:
I think we should set `brcnv-iface` up last anyway right?
So we should probably keep the `nmcli c up brcnv-iface` and fix the redundancy like so:
```
for c in $(nmcli c show | awk '{print $1}' | grep ovs); do nmcli c up $c; done
nmcli c up brcnv-iface
```

what do you think?

Reply:
Or even better, if we agree to use consistent slave connection names, this would be:
```
for c in ovs-bridge-brcnv ovs-slave-bond0 ovs-slave-brcnv-port ovs-slave-1 ovs-slave-2 brcnv-iface; do nmcli c up $c; done
```

Reply:
Fine by me, I mean, I would like to set think more consistent and not let the system decide. 

PS, 
I would use something more generic to get all OVS connections and start them
for c in (nmcli c | grep ovs ); do nmcli c up $c; done (Something link this) 
But the idea is the same. 

@phoracek what do you say?

Reply:
I like the idea of being explicit here, with profile names and their order. Leaves less space for unexpected behavior.

Reply:
Done